pom.xml

```diff
@@ -107,78 +105,70 @@
+			<groupId>io.fabric8</groupId>
+			<artifactId>kubernetes-client</artifactId>
+			<version>5.3.2</version>
 		</dependency>
 		<dependency>
-		    <groupId>io.fabric8</groupId>
-		    <artifactId>tekton-client</artifactId>
-		    <version>5.5.0</version>
+			<groupId>io.fabric8</groupId>
+			<artifactId>tekton-client</artifactId>
+			<version>5.5.0</version>
 		</dependency>
 		<dependency>
-		    <groupId>io.fabric8</groupId>
-		    <artifactId>tekton-model-v1beta1</artifactId>
-		    <version>5.5.0</version>	      
+			<groupId>io.fabric8</groupId>
+			<artifactId>tekton-model-v1beta1</artifactId>
+			<version>5.5.0</version>
 		</dependency>
 		<dependency>
-		    <groupId>io.fabric8</groupId>
-		    <artifactId>tekton-model-triggers</artifactId>
-		    <version>5.5.0</version>
+			<groupId>io.fabric8</groupId>
+			<artifactId>tekton-model-triggers</artifactId>
+			<version>5.5.0</version>
 		</dependency>
 		<dependency>
-		    <groupId>io.fabric8</groupId>
-		    <artifactId>knative-model</artifactId>
-		    <version>5.5.0</version>
+			<groupId>io.fabric8</groupId>
+			<artifactId>knative-model</artifactId>
+			<version>5.5.0</version>
 		</dependency>
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-devtools</artifactId>
 			<optional>true</optional>
 		</dependency>
 		<dependency>
-			<groupId>io.springfox</groupId>
-			<artifactId>springfox-swagger2</artifactId>
-			<version>2.9.2</version>
-			<exclusions>
-				<exclusion>
-					<groupId>com.google.guava</groupId>
-					<artifactId>guava</artifactId>
-				</exclusion>
-			</exclusions>	
-		</dependency>
-		<dependency>
-		    <groupId>com.google.guava</groupId>
-		    <artifactId>guava</artifactId>
-		    <version>28.0-jre</version>
-		</dependency>
-		<dependency>
-		    <groupId>com.github.fge</groupId>
-		    <artifactId>json-patch</artifactId>
-		    <version>1.9</version>
+			<groupId>com.github.fge</groupId>
+			<artifactId>json-patch</artifactId>
+			<version>1.9</version>
 		</dependency>
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
 		<dependency>
-            <groupId>com.github.tomakehurst</groupId>
-            <artifactId>wiremock</artifactId>
-            <version>2.19.0</version>
-            <scope>test</scope>
-        </dependency>
-
+			<groupId>com.github.tomakehurst</groupId>
+			<artifactId>wiremock-jre8</artifactId>
+			<version>2.32.0</version>
+			<scope>test</scope>
+		</dependency>
 		<dependency>
 			<groupId>org.apache.httpcomponents</groupId>
 			<artifactId>httpclient</artifactId>
 		</dependency>
-
-        <dependency>
-           <groupId>com.google.code.gson</groupId>
-           <artifactId>gson</artifactId>
-           <version>2.8.8</version>
-        </dependency>
+		<dependency>
+			<groupId>com.google.code.gson</groupId>
+			<artifactId>gson</artifactId>
+			<version>2.8.9</version>
```

Comment:
It seems that the `version` is inherited here, so you can delete the version from the artifact.